.ci/release/Jenkinsfile

```diff
@@ -17,8 +17,11 @@ pipeline {
     NOTIFY_TO = 'build-apm+apm-agent-java@elastic.co'
     BRANCH_SPECIFIER = "${params.branch_specifier}"
     SUFFIX_ARN_FILE = 'arn-file.md'
-    //RELEASE_AWS_LAMBDA_VERSION = '-ver-1-29-0'
-    //RELEASE_VERSION = '1.29.0'
+    JOB_GCS_CREDENTIALS = 'internal-ci-gcs-plugin'
+    RELEASE_AWS_LAMBDA_VERSION = '-ver-1-30-0'
+    RELEASE_VERSION = '1.30.0'
+    RELEASE_TAG = "v1.30.0"
+    BRANCH_DOT_X = '1.x'
```

Comment:
These env variables are generated in a stage, but as we disabled the executed stages, let's enforce the version manually, to be removed later on